lxd/node/config.go

```diff
@@ -169,16 +169,56 @@ var ConfigSchema = config.Schema{
 	"storage.images_volume":  {},
 }
 
+func validateCoreHTTPSAddress(value string) error {
+	if value == "" {
+		return nil // Deleting entry
+	}
+
+	host, _, err := net.SplitHostPort(value)
+	if err != nil {
+		return errors.Wrap(err, "Address not in form of <HOST>:<PORT>")
+	}
```

Comment:
I thought we currently supported just `::` or any other address and it would then default to the usual port number (8443 or 8444 in this case).